axelor-sale/src/main/java/com/axelor/apps/sale/service/configurator/ConfiguratorServiceImpl.java

```diff
@@ -216,21 +223,33 @@ public void generateProduct(
       Long saleOrderId)
       throws AxelorException {
 
+    Product product = new Product();
+    regenerateProduct(configurator, product, jsonAttributes, jsonIndicators, saleOrderId);
```

Comment:
it feels a bit strange to "regenerate product" to generate a product. Maybe we should rename the method "fillProductFields" or something like this.